File: domain/src/main/java/com/aceplus/domain/model/LoginResponse.java
package com.aceplus.domain.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by haker on 2/3/17.
 */
public class LoginResponse {

    @SerializedName("aceplusStatusCode")
    public String aceplusStatusCode;
    @SerializedName("aceplusStatusMessage")
    public String aceplusStatusMessage;

    public String getAceplusStatusCode() {
        return aceplusStatusCode;
    }

    public void setAceplusStatusCode(String aceplusStatusCode) {
        this.aceplusStatusCode = aceplusStatusCode;
    }

    public String getAceplusStatusMessage() {
        return aceplusStatusMessage;
    }

    public void setAceplusStatusMessage(String aceplusStatusMessage) {
        this.aceplusStatusMessage = aceplusStatusMessage;
    }
}

File: domain/src/main/java/com/aceplus/domain/networkresponse/LoginResponse.kt
package com.aceplus.domain.networkresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LoginResponse<T> : BaseResponseVO<T>() {

    @SerializedName("user_id")
    @Expose
    private var userId: String? = null

    @SerializedName("route")
    @Expose
    private var route: Int? = null

    @SerializedName("tablet_id")
    @Expose
    private var tabletKey: Int? = null

    @SerializedName("max_id")
    @Expose
    private var maxKey: Int? = null

}